dapps/digital-twin-detail/src/components/DigitalTwinDetail.ts

```diff
@@ -21,18 +21,22 @@ import Component, { mixins } from 'vue-class-component';
 import { DigitalTwinTemplate } from '@evan.network/api-blockchain-core';
 import { DAppTwin, dispatchers } from '@evan.network/digital-twin-lib';
 import { downloadObject } from '@evan.network/ui';
-import { EvanComponent } from '@evan.network/ui-vue-core';
+import { EvanComponent, DbcpFormComponentClass, SwipePanelComponentClass } from '@evan.network/ui-vue-core';
 
 
 @Component
 export default class DigitalTwinDetailComponent extends mixins(EvanComponent) {
   /**
    * Show loading symbol
    */
-  loading = true;
+  duplicating = false;
   exporting = false;
   favoriteLoading = false;
+  loading = true;
 
+  /**
+   * Is the current twin a favorite?
+   */
```

Comment:
we once should talk about the sense of redundant comment blocks ... it's really obvious actually what this var represents.

Reply:
True story for this parameter. Often i miss some small comments for anything.